.drone.star

```diff
@@ -552,22 +495,18 @@ def uiTests(ctx):
     for item in default:
         params[item] = config["uiTests"][item] if item in config["uiTests"] else default[item]
 
-    suiteNames = config["uiTests"]["suites"].keys()
-
     if len(params["debugSuites"]) != 0:
         suiteNames = params["debugSuites"]
-
-    return [uiTestPipeline(ctx, suiteName, params["filterTags"]) for suiteName in suiteNames]
+        return [uiTestPipeline(ctx, suiteName, params["filterTags"]) for suiteName in suiteNames]
+    return [uiTestPipeline(ctx, None, params["filterTags"])]
 
 def uiTestPipeline(ctx, suiteName, filterTags, storage = "ocis", accounts_hash_difficulty = 4):
-    suites = config["uiTests"]["suites"]
-    paths = ""
-    suite = suites[suiteName]
-    if type(suite) == "list":
-        for path in suite:
-            paths = paths + "tests/acceptance/features/" + path + " "
```

Comment:
Note: the ability to group multiple suites into pipelines is being remove here. That is OK.
The developer can still specify individual suites, in `debugSuites`, and run each suite in its own pipeline. That is enough behavior for developers who want to run a whole suite or suites to double-check the results.

There is another issue to be done after this, that will let all the web UI tests be run easily - so that will be a different feature.

Reply:
true, debug suites need not that complex.